src/components/EditCoursePage/index.jsx

```diff
@@ -277,6 +277,8 @@ class EditCoursePage extends React.Component {
       ],
       start_date: courseData.additional_metadata.start_date,
       end_date: courseData.additional_metadata.end_date,
+      // eslint-disable-next-line max-len
+      display_on_org_page: courseData.additional_metadata.display_on_org_page !== undefined ? courseData.additional_metadata.display_on_org_page : true,
```

Comment:
nit:
```suggestion
      display_on_org_page: courseData.additional_metadata.display_on_org_page || true,
```

Reply:
I did this first. But in the case of an unchecked box (False), this condition evaluates to True.

Reply:
Ah! good point

Reply:
Ah, I was not aware of this operator.